### DISCUSSION TITLE
make this a tab and add a curve window

### DISCUSSION THREAD

**Discussion by gsgoldma** [ljleb/prompt-fusion-extension#41] [Ideas]
you could graphically place your prompts and then maybe when youre finished, it would generate the prompt code that could be automatically applied to the prompt input window. I know some extensions just automatically add prompt information like stylepile and unprompted.

hakuimg uses a tone graph now and I've seen some prompt scheduling curves for animation extensions. It would help people that don't save the syntax or don't want to learn it and make it more easily accessible.

**Comment by ljleb** [comment]
This is a cool idea! Do you have a practical ui design you think would work well with interpolation?

I don't think we can use a 2D cartesian graph for this as in theory we would need an arbitrary number of dimensions. I'll think about how to represent it and take inspiration from other extensions.

Thanks for sharing!
